src/pages/Patch.tsx

```diff
@@ -92,7 +92,12 @@ const PatchCore: React.FC = () => {
         hasData={!!patch}
         title={description || `Patch ${get(patch, "patchNumber")}`}
         badge={<PatchStatusBadge status={status} />}
-        buttons={<ActionButtons isPatchOnCommitQueue={isPatchOnCommitQueue} />}
+        buttons={
+          <ActionButtons
+            canEnqueueToCommitQueue={get(patch, "canEnqueueToCommitQueue")}
```

Comment:
Since this pr was last made we decided to start phasing out using `get()` from lodash.

We can instead do `patch?.canEnqueueToCommitQueue`

Reply:
neat. 
done